forUncrustifySources.cfg

```diff
@@ -42,6 +42,7 @@ indent_macro_brace                        = true
 indent_member                             = 3
 indent_sparen_extra                       = 0
 indent_with_tabs                          = 0
+indent_oc_inside_msg_sel                  = false
```

Comment:
as this option will be never used for uncrustify itself, please put the line as comment with the other ones.
Please, make the change(s) at the next PR